client/src/stories/Input.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Input from '../components/Input';
+
+export default {
+  title: 'Input',
+};
+
+export const Name = () => <Input type="text" size="Name" maxLength={14} />;
```

Comment:
Maybe u could also add a placeholder here :)

Reply:
I could, but there will be a text underneath that says 'type in your name' ^^ maybe I change that in the making, whatever looks better.